docs/design_notes/lift.md

```diff
@@ -167,9 +167,11 @@ It is unclear whether a similar general purpose transform could be defined if li
 
 ### Linen
 
-The lifting module does not know about the Linen `Module` API. Instead it operates directly on instances of `flax.core.Scope`.
+The lifting module does not know about the Linen `Module` API.
+Instead it operates directly on instances of `flax.core.Scope`.
 A `Scope` instance contains the variables and PRNG sequences of a `Module`.
-Each module has a `Scope` instance linked if it is bound to a parent or it was created using `init` or `apply`.
+Each `Module` instance has a `Scope` instance in the `.scope` field if it has a parent or it was created using `init` or `apply`.
+Typically, the top-level `Module` instance on which you call `init` or `apply` is the only `Module` instance that does not have a `Scope` bound to it.
```

Comment:
Suggestion (nit):

- Because there are several moving parts and new material for new users, I suggest the following to help the audience digest this:

```suggestion
Typically, the top-level `Module` instance—on which you call `init` or `apply`—is the only `Module` instance that does not have a `Scope` bound to it.
```

LMKWYT 👍

Reply:
Agree